dev/lint-python

```diff
@@ -82,6 +82,25 @@ else
     rm "$PYCODESTYLE_REPORT_PATH"
 fi
 
+# stop the build if there are Python syntax errors or undefined names
+flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
+flake8_status="${PIPESTATUS[0]}"
+# exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
+flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

Comment:
From https://spark.apache.org/contributing.html our max line length should be 100

Reply:
And to be clear, this one is just to output all of the errors/warnings not actually stop the build, thats the one above? I think this might get confusing to folks who have flake8 stop the build and then think they need to fix everything here. What do you think?